ext/featurefmt/driver.go

```diff
@@ -46,29 +46,33 @@ type Lister interface {
 	//
 	// Filenames must not begin with "/".
 	RequiredFilenames() []string
+
+	// Version provides the detector's version.
+	Info() database.Detector
```

Comment:
Please review this design, the detectors should define this Info interface to return the metadata i.e. the name, version, and type of the detector.